pkg/parser/sysl.wbnf

```diff
@@ -38,7 +38,13 @@ type_decl -> table | facade | alias | union;
 
 table -> mode=("!table"|"!type") NAME attribs?  ":" COMMENT*
      %!Indented(COMMENT_NO_NL | table_row COMMENT_NO_NL?) {
-        table_row -> (NAME array_size? "<:") type=(collection_type|type_spec) optional="?"? attribs? (":" annotations)?;
+       table_row -> COMMENT_NO_NL
```

Comment:
nitpick: this line is indented on char less than it was before and does not align with the `table_field` line.

Reply:
added the space